psydac/fem/vector.py

```diff
@@ -113,17 +113,22 @@ def eval_fields(self, grid, *fields, weights=None, npts_per_cell=None):
             Fields to evaluate.
 
         weights : psydac.fem.basic.FemField or None, optional
-            Weights field.
+            Weights field used to weight the basis functions thus
+            turning them into NURBS. The same weights field is used
+            for all of fields and they thus have to use the same basis functions.
 
         npts_per_cell: int, tuple of int or None, optional
             Number of evaluation points in each cell.
             If an integer is given, then assume that it is the same in every direction.
 
+        overlap : int
+            How much to overlap. Only used in the distributed context.
+            
         Returns
         -------
         List of list of ndarray
-            List of the same lengths as `fields`, containing for each field,
-            a list of `self.ldim` arrays, on for each logical coordinate.
+            List of the same lengths as `self.ldim`, containing for each logical
+            coordinate, a list of `len(fields)` arrays, on for each logical coordinate.
```

Comment:
This docstring should be improved, as done in other cases

Reply:
Done in 07a9a3d